dbms/src/Storages/FormatVersion.h

```diff
@@ -129,7 +129,7 @@ inline static const StorageFormatVersion STORAGE_FORMAT_V5 = StorageFormatVersio
     .identifier = 5,
 };
 
-inline StorageFormatVersion STORAGE_FORMAT_CURRENT = STORAGE_FORMAT_V4;
+inline StorageFormatVersion STORAGE_FORMAT_CURRENT = STORAGE_FORMAT_V5;
```

Comment:
Is it set only for test?

Reply:
Yes, just set it to run some test. Changed it back now.